spec/aptible/cli/subcommands/db_spec.rb

```diff
@@ -264,6 +264,120 @@ def expect_provision_database(create_opts, provision_opts = {})
     end
   end
 
+  describe '#db:tunnel (rds databases)' do
```

Comment:
this test needs to be reviewed mroe closely. i think it doesn't behave quite the way i want and is overly passing. i used AI to help but I think it's overly friendly to passage